modules/installing-with-usb-media.adoc

```diff
@@ -0,0 +1,29 @@
+// This is included in the following assemblies:
+//
+// installing_on_prem_assisted/assisted-installer-installation.adoc
+
+:_content-type: PROCEDURE
+[id="installing-with-usb-media_{context}"]
+= Creating a bootable ISO image on a USB drive
+
+You can install software using a bootable USB drive that contains an ISO image. Booting the server with the USB drive prepares the server for the software installation.
```

Comment:
We had a couple of users that expected the booting to actually start the installation as well
I would put even more emphasizing about it just registering the nodes to the cluster, enabling further pre-flight checks and configurations to take place afterwards